torch/csrc/jit/codegen/cuda/fusion.cpp

```diff
@@ -391,6 +457,7 @@ StmtNameType Fusion::registerExpr(Expr* expr) {
 
   for (Val* input : expr->inputs()) {
     assertInFusion(input, "Input to expr is invalid, ");
+    // TORCH_CHECK(!inKernelIr(input));
```

Comment:
Should this be uncommented?

Reply:
No, that's the stepping stone to the next iteration :) It would fail right now since Kernel IR expression still point to Fusion IR values.

But leaving it commented in there was sloppy on my part. The right thing to do is to remove the line and add it back in the follow up PR. I'll make the change.